registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/service/impl/ManualVerificationServiceImpl.java

```diff
@@ -678,6 +813,38 @@ private ManualAdjudicationRequestDTO prepareManualAdjudicationRequest(MessageDTO
 
 		return req;
 	}
+	
+	private List<ReferenceURL> addReferenceURLs(String id,InternalRegistrationStatusDto registrationStatusDto) throws Exception {
+		List<ReferenceURL> referenceURLs=new ArrayList<>();
+		if( registrationStatusDto.getStatusCode().equalsIgnoreCase(RegistrationStatusCode.PROCESSED.name())) {
+			ReferenceURL referenceURL=new ReferenceURL();
+			referenceURL.setSource(ID_REPO);
+			referenceURL.setStatus(PROCESSED);
+			referenceURL.setURL(
+					getDataShareUrlfromIdRepo(id));
+			referenceURLs.add(referenceURL);
+		}
+		if(! registrationStatusDto.getStatusCode().equalsIgnoreCase(RegistrationStatusCode.PROCESSED.name()))
+			 {
+			ReferenceURL referenceURL=new ReferenceURL();
+			referenceURL.setSource(PACKET);
+			referenceURL.setStatus(IN_PROGRESS);
```

Comment:
set status from registrationStatusDto.getStatusCode() here

Reply:
I have changed the code